### PR TITLE
added support tuya motion sensor TS0202 _TZ3000_lltemgsf

### PR DESCRIPTION
**Pull request by d51x**

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -849,6 +849,20 @@ module.exports = [
         toZigbee: [],
         exposes: [e.occupancy(), e.battery_low(), e.tamper(), e.battery()],
     },
+    {
+        fingerprint: [{modelID: 'TS0202', manufacturerName: '_TZ3000_lltemgsf'}],
+        model: 'TS0202_2',
```
Comment (Koenkk) [inline devices/tuya.js:854]: This can be merged with `TS0202_1`

Comment (d51x) [reply devices/tuya.js:854]: ok, but _TZ3000_lltemgsf dosn't have battery and tamper
only occupancy, battery_low

Comment (Koenkk) [reply devices/tuya.js:854]: @groenmarsmannetje does the `tamper` work for your TS0202_1 motion sensor?

Comment (groenmarsmannetje) [reply devices/tuya.js:854]: @koenkk I do not think so. Will double check

Comment (groenmarsmannetje) [reply devices/tuya.js:854]: I checked and the tamper sensor doesn’t turn on when I open the device to remove battery.

Comment (Koenkk) [reply devices/tuya.js]: Thanks!

**Comment by yormedia** [comment]
Hi, I think you removed a bit to much. Now the battery level percentage is also removed.

**Pull request merged**